Sources/MapboxNavigation/StatusView.swift

```diff
@@ -52,30 +52,57 @@ public class StatusView: UIControl {
     `Status` is a struct which stores information to be displayed by the `StatusView`
      */
     public struct Status {
+        /**
+        `identifier` is a unique string identifier for a `Status`
```

Comment:
FYI, it isn’t necessary to explicitly name the property in the property’s own documentation comment. For example, this could say “A string that uniquely identifies this status.”